src/config/Config.cpp

```diff
@@ -265,4 +265,17 @@ void Config::SetWindowBackend(WindowBackend backend) {
     }
 }
 
+void Config::RegisterConfigVersion(std::shared_ptr<ConfigVersionUpdater> version) {
+    mVersions.push_back(version);
+}
```

Comment:
the naming here feels a little confusing, maybe `mVersionUpdaters` and `versionUpdater` instead?

Reply:
I agree, will be renaming them to mVersionUpdaters and RegisterConfigVersionUpdater.

Reply:
Fixed to the new names.